src/js/navIcons.js

```diff
@@ -47,12 +47,15 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
             that.locate("confirmedIndicator").addClass(that.options.styles.show);
         }
     };
-
+    
```

Comment:
Yes, probably worth to dig up a linting config for trailing spaces. Luckily, the "sublime text" editor I use has a plugin that automatically removes all trailing spaces at every save.